### PR TITLE
Add files for docker container building

### PR DESCRIPTION
**Pull request by neubig**
This PR adds:
1. `Dockerfile`s to build the frontend and backend
2. A buildspec.yml file that can be used for specifying build behavior for AWS

### CODE REVIEW

File: buildspec.yml
Comment (Sparkier) [inline buildspec.yml:1]: Do we want to add some description of how to use this to our `DEVELOPMENT.md`?

Comment (Sparkier) [reply buildspec.yml:1]: Including required env variables probably.

**Review by Sparkier — APPROVED** [review]

**Comment by cabreraalex** [comment]
This doesn't include anything for the database correct?

I've been thinking what the best development setup would be, maybe a docker file or docker compose to run the database and backend...

File: buildspec.yml
Comment (neubig) [reply buildspec.yml:1]: Added to development.md

**Comment by neubig** [comment]
> I've been thinking what the best development setup would be, maybe a docker file or docker compose to run the database and backend...

I see that a new issue is created for this, so I'll cede this to another PR: https://github.com/Sparkier/zeno_hub/issues/73

**Comment by neubig** [comment]
@Sparkier : I don't have write access to this repo, so please merge this in if it looks OK.

**Pull request merged**
